app.js

```diff
@@ -13,13 +13,43 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
+hbs.registerPartials(path.join(__dirname, 'views/partials'));
+
+
+
 
-// ...
 
 // Add the route handlers here:
 
 app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+  .getBeers()
+  .then((beersFromApi) => {
+    console.log('Beers from the database: ', beersFromApi)
+    res.render('beers', {beersFromApi});
+  })
+
+  .catch((error) => {
+    console.log(error)
+  });
+  
+  //console.log("chin chin")
+});
+
+app.get('/random-beers', (req, res) => {
+  
+  const randomBeer = punkAPI.getRandom()
+
+  randomBeer.then(beer => {
+    console.log(beer[0].name)
+    res.render('random-beers', {beer});
```

Comment:
since you already know the array only contains one element you can instead of sending the beer array you can send only the first element to the hbs.
`res.render('random-beers', {beer: beer[0]});`